docs/docs/85-SampleGenerator.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: docs
+title: Creating A Sample Generator
+---
+
+# Creating A Sample Generator
+
+## What Is A Sample Generator?
+
+Glad you asked! A Sample Generator generates Samples... Ok, that isn't helpful. A Sample Generator is an object in Refocus that defines how you collect Samples for a defined set of Subjects and Aspects. Creating a Sample Generator is easy and allows you to immediately stream your real-time Sample data into Refocus.
+
+## Prerequisites
+
+1. Have an instance of Refocus running [link](https://salesforce.github.io/refocus/docs/03-quickstartheroku.html)
+2. Have at least 1 collector deployed that is able to reach the data source you will be collecting from [link](https://github.com/salesforce/refocus-collector#installation)
+3. Have a Sample Generator Template for the information you're trying to collect [link](https://github.com/salesforce/refocus-sample-generator-template-utils#refocus-sample-generator-template-utils-refocus-sample-generator-template-utilities)
+
+## 5 Main Concepts You Need To Understand
+
+### Subjects 
+
+This tells the Sample Generator what Subjects you will be monitoring. You can define Subjects by EITHER
+passing in a list of Subject Absolute Paths into the “subjects” array
+OR
+by defining a query in the “subjectQuery” string. The “subjectQuery” is inserted in the subjects API endpoint like so /v1/subjects/{subjectQuery} 
```

Comment:
should include example of what a subject query might look like, e.g. `?absolutePath=NorthAmerica.UnitedStates.California.*&tags=Indigo`